stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -15,18 +16,28 @@ OIDC auth service:
 
 ## How to use
 
-This configuration should be included as part of kubeflow deployment. After a successful Kubeflow deployment, you will find no pod running in the statefulset
-`authservice` in namespace `istio-system`. It will need to fill in valid 
-configuration from the environment variables via either web console or CLI with parameters from AppID instance:
-* `OIDC_PROVIDER` - fill in the value of `oAuthServerUrl`
-* `CLIENT_ID` - fill in the value of `clientId`
-* `CLIENT_SECRET` - fill in the value of `secret`
-* `REDIRECT_URL` - fill in `https://<hostname-of-the-route-istio-ingressgateway>/login/oidc`
+1. Create the namespace `istio-system` if you don't have it:
+```SHELL
+kubectl create namespace istio-system
+```
+2. Create a secret prior to kubeflow deployment by filling parameters accordingly:
+```SHELL
+kubectl create secret generic appid-application-configuration -n istio-system \
+  --from-literal=clientId=<clientId> \
+  --from-literal=secret=<secret> \
+  --from-literal=oAuthServerUrl=<oAuthServerUrl> \
+  --from-literal=oidcRedirectUrl=https://<routeFQDN>/login/oidc
+```
+* `<oAuthServerUrl>` - fill in the value of `oAuthServerUrl`
+* `<clientId>` - fill in the value of `clientId`
+* `<secret>` - fill in the value of `secret`
+* `<routeFQDN>` - fill in the public endpoint of istio ingressgateway.
 
 Notice that the environment variable `REDIRECT_URL` should be updated with the
 actual FQDN of public endpoint of istio ingressgateway, either via ingress or
 route. And please keep the path to `/login/oidc`.
 
-After updating the statusfulset `authservice`, you will need to add the value of
-`REDIRECT_URL` to the AppID instance under the _Authentication Settings_ of the _Manage Authentication_ menu. Or AppID won't redirect authenticated requests
+After deploying Kubeflow successfully, you will need to add the value of
+`oidcRedirectUrl` to the IBM Cloud AppID instance under the _Authentication 
```

Comment:
instead of `oidcRedirectUrl` without noting what it refers to, may just use `https://<routeFQDN>/login/oidc`?